docs/app/pages/charts/sections/spark-charts/spark-chart-ng1/spark-chart-ng1.component.ts

```diff
@@ -6,7 +6,7 @@ import { ICodePen } from '../../../../../interfaces/ICodePen';
 @Component({
     selector: 'uxd-charts-spark-chart-ng1',
     templateUrl: './spark-chart-ng1.component.html',
-    styleUrls: ['./spark-chart-ng1.component.less'],
+    styleUrls: ['./../spark-charts.less'],
```

Comment:
I don't think we should do this. Add common documentation styles to the docs/styles.less file.